hazelcast/src/main/java/com/hazelcast/map/impl/operation/PutAllOperation.java

```diff
@@ -68,6 +68,14 @@ public PutAllOperation(String name, MapEntries mapEntries, boolean initialLoad)
     }
 
     @Override
+    public void beforeRun() throws Exception {
+        super.beforeRun();
+        if (shouldWanReplicate()) {
```

Comment:
Seems like they are same. It's possibly a leftover from times before `isWanReplicationEnabled` method is introduced.